test/e2e/requirements.txt

```diff
@@ -32,3 +32,4 @@ six==1.15.0
 toml==0.10.2
 urllib3==1.26.4
 websocket-client==0.58.0
+black==20.*
```

Comment:
black isn't a requirement for the test code itself. It's a tool used in testing.

Reply:
yes, for linting, going forward all testing PRs in this repo will be formatted using this tool and in future, we plan to add it as a check in PR builds

Reply:
How about adding the pytest-black plugin to check for zero diffs as an integ test already ? or is there some reason we dont want this in our integration tests ?

Reply:
couldn't find a way to run this tool as part of integ test, it looks like a CLI tool

so we can use it when we have PR builds to check if it returns any diff